packages/circuit-ui/components/AspectRatio/__snapshots__/AspectRatio.spec.tsx.snap

```diff
@@ -1,28 +1,9 @@
 // Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
 
 exports[`AspectRatio > should render with default styles 1`] = `
-.circuit-0 {
-  display: block;
-  position: relative;
-  overflow: hidden;
-  height: auto;
-  width: 100%;
-}
```

Comment:
I removed these default styles when no aspect-ratio is provided. The result should be the same.